accounts/rpcserver.go

```diff
@@ -137,7 +137,46 @@ func (s *RPCServer) UpdateAccount(ctx context.Context,
 func (s *RPCServer) UpdateBalance(ctx context.Context,
 	req *litrpc.UpdateAccountBalanceRequest) (*litrpc.Account, error) {
 
-	return nil, fmt.Errorf("not implemented")
+	var (
+		isAddition bool
+		amount     lnwire.MilliSatoshi
+	)
+
+	switch reqType := req.GetUpdate().(type) {
+	case *litrpc.UpdateAccountBalanceRequest_Add:
+		log.Infof("[addbalance] id=%s, label=%v, amount=%d",
```

Comment:
what do you think of using "credit/debit" instead of "add/deduct"?

Reply:
Ah yeah definitely! I was struggling with naming, and agree that credit & debit is better.